### DISCUSSION TITLE
mod_jea_emphasis download

### DISCUSSION THREAD

**Discussion by Enr62** [JoomlaEstateAgency/com_jea#39] [Q&A] [answered]
Hi, where can I download an updated "mod_jea_emphasis.zip" package for Joomla 4? Thanks for your help

**Comment by ilhooq** [comment, marked as answer]
[Here](https://github.com/JoomlaEstateAgency/mod_jea_emphasis/archive/refs/heads/master.zip)

**Comment by Enr62** [comment]
Thank you so much! Grazie 1000!
